pepper-apis/configure.rb

```diff
@@ -1,15 +1,25 @@
 #!/usr/bin/ruby -w
+# Docker Image used to call out to generate config files
+$dsde_toolbox_image_name = "broadinstitute/dsde-toolbox:latest"
+
+# If USE_DOCKER evaluates to false, script will try to call consul-template directly
+# instead of using Docker image built in scripts
+$use_docker = ENV.fetch("USE_DOCKER", "true") == "true"
+
+$gae = ENV.fetch("GAE", "false")
 
 $version = ENV.fetch("VERSION") { |_|
   puts "VERSION not set"
   exit 1
 }
 
-$env = ENV.fetch("ENV") { |_|
-  puts "ENV not set"
+$env = ENV.fetch("ENVIRONMENT") { |_|
+  puts "ENVIRONMENT var not set"
   exit 1
 }
```

Comment:
The name ENV was problematic in either consult-template (I don't recall what the problem was at this point)